README.md

```diff
@@ -451,6 +451,7 @@ For more information about inheriting configuration from a gem please check
 
 * Avoid magic numbers. Use a constant and give it a useful name.
 
+* Avoid nomenclature that has (or could be interpretted to have) discriminatory origins.
```

Comment:
Since this would apply to all programming languages used at Shopify, is there any other shared guide where this could be covered?

(also, typo in 'interpretted').